tests/browser_test.go

```diff
@@ -343,3 +343,27 @@ func TestMultiConnectToSingleBrowser(t *testing.T) {
 	err = p2.Close(nil)
 	require.NoError(t, err, "failed to close page #2")
 }
+
+func TestCloseContext(t *testing.T) {
+	t.Parallel()
+
+	t.Run("close_context", func(t *testing.T) {
+		t.Parallel()
+
+		tb := newTestBrowser(t)
+		_, err := tb.NewContext(nil)
+		require.NoError(t, err)
+
+		err = tb.CloseContext()
+		require.NoError(t, err)
+
+		assert.Nil(t, tb.Context())
```

Comment:
This test checks the error behavior well. But, the test will pass even if `CloseContext` does nothing and `NewContext` sets a `nil` context (and returns a `nil` error). Other tests already test this functionality extensively, but it's better for this test to do this here without depending on others. It'd also be more expressive in terms of "tests as documentation".

```suggestion
		assert.NotNil(t, tb.Context())
		
		err = tb.CloseContext()
		require.NoError(t, err)

		assert.Nil(t, tb.Context())
```

Reply:
Fixed in [421c38c](https://github.com/grafana/xk6-browser/pull/1074/commits/421c38c1d373699d98837a6f9a4402391af800d2).